sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java

```diff
@@ -170,27 +170,32 @@ private void saveContactsFromLineListing(LineListingLayout lineListingForm, Link
 	}
 
 	public void openLineListingWindow(EventDto eventDto, Set<EventParticipantIndexDto> eventParticipantIndexDtos) {
-		if (eventParticipantIndexDtos == null || eventParticipantIndexDtos.isEmpty()) {
-			return;
-		}
+		if (eventParticipantIndexDtos.size() == 0) {
+			new Notification(
```

Comment:
The whole message block could have been added to the pre-existing if clause without adding an else statement.